mula/scheduler/storage/filters/functions.py

```diff
@@ -27,11 +27,25 @@ def apply_filter(entity: DeclarativeBase, query: Query, filter_request: FilterRe
     for operator in filter_request.filters:
         expressions = []
         for filter_ in filter_request.filters[operator]:
+            if not hasattr(entity, filter_.column):
```

Comment:
From here you're using `filter_.column` a few times, perhaps extract it to a variable `column`?